config/default.yaml

```diff
@@ -15,7 +15,7 @@ services:
     systemUsername: 'dummy_user'
     systemPassword: 'dummy_password'
   fis:
-    url: 'http://fis-cos-api-pr-80.service.core-compute-preview.internal'
+    url: 'https://fis-cos-api-pr-80.service.core-compute-preview.internal'
```

Comment:
you should make these env vars and handle as env vars in charts and locals. This is not sustainable.